front/app/containers/Admin/settings/registration/CustomFieldSettings/index.tsx

```diff
@@ -118,8 +122,9 @@ class CustomFields extends Component<Props & WrappedComponentProps, State> {
 
       if (window.confirm(deleteMessage)) {
         this.setState({ itemsWhileDragging: null, isProcessing: true });
-        deleteUserCustomField(customFieldId).then(() => {
+        deleteUserCustomField(customFieldId).then(async () => {
           this.setState({ isProcessing: false });
+          await Promise.all([streams.reset(), resetQueryCache()]);
```

Comment:
Seems a bit overkill to invalidate all the cache... maybe we should just invalidate stuff related to custom fields?

Reply:
@luucvanderzee Sounds good - would that be with the "fetchAllWith" then that I mentioned in the PR description? Or is there another way to invalidate streams? :thinking: